R/visualization/plotSummaryScatter.R

```diff
@@ -57,6 +57,26 @@ plotSummaryScatter <- function(scores) {
 
     `%>%` <- magrittr::`%>%`
 
+    methods_pal <- c(
+        "BBKNN"          = "#5A5156",
+        "Conos"          = "#E4E1E3",
+        "trVAE"          = "#F6222E",
+        "scVI"           = "#FE00FA",
+        "ComBat"         = "#16FF32",
+        "Harmony"        = "#3283FE",
+        "LIGER"          = "#FEAF16",
+        "Scanorama"      = "#B00068",
+        "Seurat v3 CCA"  = "#1CFFCE",
+        "Seurat v3 RPCA" = "#90AD1C",
+        "MNN"            = "#2ED9FF",
+        "fastMNN"        = "#DEA0FD",
+        "scGen*"         = "#AA0DFE",
+        "scANVI*"        = "#F8A19F",
+        "DESC"           = "#325A9B",
+        "SAUCIE"         = "#C4451C",
+        "Unintegrated"   = "#66B0FF"
+    )
```

Comment:
This maybe shouldn't be hardcoded either...

Reply:
Could just be added to a separate mapping file that is loaded, like the tool name conversions above.

Reply:
This is one of mine. I'll fix it after this PR is merged.